crates/iota-indexer/README.md

```diff
@@ -83,6 +83,36 @@ cargo run --bin iota-indexer -- --db-url "postgres://postgres:postgrespw@localho
 
 More available flags can be found in this [file](https://github.com/iotaledger/iota/blob/develop/crates/iota-indexer/src/lib.rs).
 
+### Backfilling of data
+
+When a new database table is introduced, backfilling may be required to populate historical data.
```

Comment:
This is not limited to new database tables. We might want to backfill a new column, or an existing column that its semantics are changed. I would therefore suggest generalizing to

```suggestion
Sometimes when the schema changes (e.g. adding a new table or column), backfilling may be required to populate historical data.
```